src/grpc.ts

```diff
@@ -280,6 +280,7 @@ export class GrpcClient {
     // To keep the existing behavior and avoid libraries breakage, we pass -1 there as suggested.
     grpcOptions['grpc.max_receive_message_length'] = -1;
     Object.keys(options).forEach(key => {
+      const value = options[key];
```

Comment:
shouldn't we add a line, along the lines of:

```js
if (!value) return
```

Or am I missing what's being fixed here.

Reply:
yeah, it's hard to stop.  let's say you have `options = { 'grpc.gcpApiConfig': { something: 42 } }`.
in line 292, the `key` becomes `gcpApiConfig`. 
in line 294, I'm was trying to get `options[gcpApiConfig]` (which never existed).